src/wikibots/inaturalist.py

```
@@ -0,0 +1,132 @@
+from typing import Any
+
+import requests
+from pywikibot import warning, info, Claim
+from pywikibot.page import BasePage
+from pywikibot.pagegenerators import SearchPageGenerator
+
+try:
+    sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/lib')
+    from lib.bot import BaseBot
+    from lib.wikidata import WikidataEntity, WikidataProperty
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)